### PR TITLE
Updated text

### PR DESCRIPTION
**Pull request by catherinepope**
# Description
- Tidied text and applied style guide.
- Improved clarity.
- Added screenshots
- Removed section on Chroma (it's a third-party tool and only works for Python)

# Reasons
https://circleci.atlassian.net/browse/DOCSTEAM-12

### CODE REVIEW

File: jekyll/_cci2/notifications.md [hunk 1]
```diff
@@ -11,85 +11,84 @@ version:
...
-
+CircleCI offers integrated email and web notifications. IRC and Slack notifications can also be configured using orbs. Slack, email, and web notifications are delivered on the succesful completion or failure of a [workflow]({{ site.baseurl }}/2.0/workflows/). IRC notifications are delivered for each job. Consider the minimal CircleCI config below:
 
 ```yaml
-version: 2
+version: 2.1
 jobs:
   build:
     docker:
       - image: cimg/<language>:<version TAG>
```
Comment (rosieyohannan) [inline jekyll/_cci2/notifications.md:21]: I feel like this syntax is probably quite confusing, shall we just use the most basic image?

File: jekyll/_cci2/notifications.md [hunk 2]
```diff
@@ -11,85 +11,84 @@ version:
...
+## Enabling web notifications
 {: #enable-web-notifications }
 
 Perform the following steps to enable web notifications:
 
-1. Go to your [CircleCI user settings](https://app.circleci.com/settings/user/notifications){:rel="nofollow"}. Enable the toggle for "Web Notifications" at the bottom of the document.
+1. Go to your [CircleCI user settings](https://app.circleci.com/settings/user/notifications){:rel="nofollow"}. Enable the toggle for "Web Notifications" at the bottom of the page.
 
-2. Your browser will ask you to confirm that you want to allow notifications. Click `Allow`. See the screenshot below for additional details:
+2. Your browser asks you to confirm that you want to allow notifications. Click `Allow`. See the screenshot below for additional details:
```
Comment (rosieyohannan) [inline jekyll/_cci2/notifications.md:68]:
```suggestion
2. Your browser asks you to confirm that you want to allow notifications. Click **Allow**. See the screenshot below for additional details:
```

File: jekyll/_cci2/notifications.md [hunk 3]
```diff
@@ -119,42 +118,38 @@ workflows:
...
 
-## Third party tools
-{: #third-party-tools }
-
-### Chroma feedback
-{: #chroma-feedback }
+## See also
 
-[Chroma Feedback](https://github.com/redaxmedia/chroma-feedback) is a command line tool in Python to turn your RGB powered hardware into an build indicator. The idea of such extreme visibility is to encourage developers to instantly repair their builds.
+- [Orbs Introduction]({{ site.baseurl }}/2.0/orb-intro/)
```
Comment (rosieyohannan) [inline jekyll/_cci2/notifications.md:155]: Maybe we could add a link to the slack orb tutorial tot his see also section too?

**Review by rosieyohannan — APPROVED** [review]
Thanks @catherinepope big improvement!

File: jekyll/_cci2/notifications.md (hunk 1 above)
Comment (catherinepope) [reply jekyll/_cci2/notifications.md]: Good idea!

**Pull request merged**